app/uk/gov/hmrc/incometaxsubscriptioneligibility/services/ControlListEligibilityService.scala

```diff
@@ -32,36 +33,46 @@ class ControlListEligibilityService @Inject()(convertConfigValuesService: Conver
                                               getControlListConnector: GetControlListConnector,
                                               auditService: AuditService
                                              ) extends FeatureSwitching {
-
-  private def eligibilityConfigParameters(year: String) = convertConfigValuesService.convertConfigValues(year)
-
   def getEligibilityStatus(sautr: String, userType: String, agentReferenceNumber: Option[String])
-                          (implicit hc: HeaderCarrier, ec: ExecutionContext, request: Request[_]): Future[EligibilityByYear] =
+                          (implicit hc: HeaderCarrier, ec: ExecutionContext, request: Request[_]): Future[EligibilityStatus] =
     if (isEnabled(StubControlListEligible)) {
-      Future.successful(EligibilityByYear(current = true, next = true))
+      Future.successful(EligibilityStatus(eligible = true, eligibleCurrentYear = true, eligibleNextYear = true))
```

Comment:
This might be something we need to consider whether we should be doing this anymore.
Could we potentially, make the call out to the control list ALL the time, then if this feature switch is on, override the eligibility status to true?